app/javascript/components/pages/key-performance-indicators/actions.js

```diff
@@ -0,0 +1 @@
+export const NUMBER_OF_CASES = "KeyPerformanceIndicators/NUMBER_OF_CASES";
```

Comment:
Our rest middleware generates a few actions based on this action. `NUMBER_OF_CASES_SUCCESS, NUMBER_OF_CASES_STARTED, NUMBER_OF_CASES_FINISHED, NUMBER_OF_CASES_FAILURE`. `NUMBER_OF_CASES_SUCCESS` will have the payload of your response from the backend. `NUMBER_OF_CASES_FAILURE` will return any errors and the other two are just loading statuses. I guess thats more of a fyi since you are mocking data right now.

Reply:
I realized this after first implementing my own actions! The info is still useful as confirmation though, thanks!